README.rst

```diff
@@ -302,6 +302,7 @@ Building DALI using Clang (experimental):
 -  ``BUILD_NVOF`` - build with ``NVIDIA OPTICAL FLOW SDK`` support (default: ON)
 -  ``BUILD_NVDEC`` - build with ``NVIDIA NVDEC`` support (default: ON)
 -  ``BUILD_NVML`` - build with ``NVIDIA Management Library`` (``NVML``) support (default: ON)
+-  ``BUILD_TYPE`` - select build configuration. Supported configurations: Debug, Release (default: Release)
```

Comment:
This is wrong: It's CMAKE_BUILD_TYPE, and it's regular CMake thing.
Here we list mostly options and custom variables we use in CMake.
Also we have 4 configurations: Debug, DevDebug, Release, RelWithDebInfo